snap/hooks/configure

```diff
@@ -133,6 +133,14 @@ then
   "$SNAP/bin/sed" -i 's@requestheader-client-ca-file=${SNAP_DATA}/certs/ca.crt@requestheader-client-ca-file=${SNAP_DATA}/certs/front-proxy-ca.crt@g' ${SNAP_DATA}/args/kube-apiserver
 fi
 
+# In 1.27 we removed the --containerd argument of cubelet. This however is used by cadvisor
```

Comment:
Shouldn't this also be in the PR for the main branch?

Reply:
@berkayoz the patch in the `configure` hook is for users following the 1.27 track. With this patch we add the `--containerd` argument. The master branch feeds the latest/edge and latest/beta channels. These channels are meant to be used for testing and ephemeral deployments so we do not need to apply this patch there. The `--containerd` will land on such deployments as a default argument.